src/components/app/TourForm.tsx

```diff
@@ -109,7 +109,7 @@ export default function TourForm ({ tour, saveHandler, type, formErrors }: TourF
                                                  features={[new TourPoint(currentTour.startLocation), new TourPoint(currentTour.endLocation)]}
                                                  type={type}/>
                         }
-                          <GpxDataLayer gpxFile={file} handleSetMarker={handleSetMarker}/>
+                          <GpxDataLayer gpxFile={file!} handleSetMarker={handleSetMarker}/>
```

Comment:
Shouldn't we fix the typing on GpxDataLayer to avoid this assertion? Because we're checking for its content anyways, so we're faking an assertion here I think 😅